main.py

```diff
@@ -2,6 +2,53 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    create_genre = [
+        "Western",
+        "Action",
+        "Dramma"
+    ]
+
+    for name in create_genre:
+        Genre.objects.create(
+            name=name
+        )
+
+    create_actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    checklist = [
+        {"field": "first_name", "index": 0},
+        {"field": "last_name", "index": 1}
+    ]
+
+    for data in create_actors:
+        actor_data = {item["field"]: data[item["index"]] for item in checklist}
+        Actor.objects.create(**actor_data)
```

Comment:
it is too complicated solution, try to simplify this 
could u create a several variables in for loop to unpack ur actors ?